tests/unit/spec/base/core-datalayer.js

```diff
@@ -25,6 +25,60 @@ describe('core-datalayer.js', function() {
         });
     });
 
+    describe('updateDataLayerPush', function() {
+
+        beforeAll(function() {
+            Mozilla.Analytics.updateDataLayerPush();
+        });
+
+        beforeEach(function() {
+           var link = '<a id="link" href="https://www.mozilla.org/en-US/firefox/new/">';
+            $(link).appendTo('body');
+            window.linkElement = $('#link')[0];
```

Comment:
Are you talking about the window.linkElement declaration? I want linkElement to be deleted and re-instantiated after every test though as I alter it during one of the tests and I want each new test to start with the same link.